package.json

```diff
@@ -5,7 +5,7 @@
     "description": "Lambda-based graphQL API for camera trap data management platform",
     "main": "src/handler.js",
     "scripts": {
-        "start": "serverless offline",
+        "start": "AWS_PROFILE=animl serverless offline --noAuth",
```

Comment:
@jue-henry, just curious, but do we need `--noAuth` here?

Reply:
We get this warning while running without it
`Warning: Serverless Offline does not support non local authorizers (arn): [object Object]`

I think this option disables the need to include `x-api-key` which actually isn't a huge deal if we test exclusively using the frontend, but this also gets rid of the warning.

Reply:
Got it!